MoneyLoverClone/Scenes/Account/AccountViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  AccountViewController.swift
+//  MoneyLoverClone
+//
+//  Created by nguyen viet hoang on 9/22/20.
+//  Copyright © 2020 Vu Xuan Cuong. All rights reserved.
+//
+
+import UIKit
+
+class AccountViewController: UIViewController {
+    
+    @IBOutlet weak var imgAccount: UIImageView!
```

Comment:
```suggestion
    @IBOutlet private weak var imgAccount: UIImageView!
```
các biến outlet bên dưới cũng thế